samples/cypress/dashboards/chart-interaction/demo.mjs

```diff
@@ -1,23 +1,29 @@
-import CSVConnector from '../../../../code/es-modules/Data/Connectors/CSVConnector.js';
-import Board from  '../../../../code/es-modules/Dashboards/Board.js';
 import PluginHandler from  '../../../../code/es-modules/Dashboards/PluginHandler.js';
 import Highcharts from  '../../../../code/es-modules/masters/highcharts.src.js';
 import HighchartsPlugin from  '../../../../code/es-modules/Extensions/DashboardPlugins/HighchartsPlugin.js';
+import Dashboards from '../../../../code/es-modules/masters/dashboards.src.js';
 
 HighchartsPlugin.custom.connectHighcharts(Highcharts);
 PluginHandler.addPlugin(HighchartsPlugin);
 
-// A shared connector
-const connector = new CSVConnector({
-    csv: `$GME,$AMC,$NOK
-        4,5,6
-        1,5,2
-        41,23,2`,
-    firstRowAsNames: true
-});
-connector.load();
 
-const board = new Board('container', {
+const board = Dashboards.board('container', {
```

Comment:
Am I correct that the outcome of the `new Dashboard` would be different than the one using the `board` factory function?
It looks like the `init` is only called inside that function.

Reply:
to be discussed on workshops